Description: pass INSTALLDIRS=vendor to Makefile.PL to get stuff into the right place
Origin: vendor
Forwarded: not-needed
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2014-07-08

--- a/swig/perl/Makefile
+++ b/swig/perl/Makefile
@@ -7,7 +7,7 @@ build: Makefile_Geo__GDAL
 	$(MAKE) -f Makefile_Geo__OSR
 
 Makefile_Geo__GDAL:
-	perl Makefile.PL INSTALL_BASE=$(INST_PREFIX)
+	perl Makefile.PL INSTALLDIRS=vendor
 
 test: build
 	$(MAKE) -f Makefile_Geo__GDAL test
--- a/swig/perl/GNUmakefile
+++ b/swig/perl/GNUmakefile
@@ -30,7 +30,7 @@ build: gdal_wrap.cc Makefile_Geo__GDAL
 gdal_wrap.cc: generate
 
 Makefile_Geo__GDAL:
-	perl Makefile.PL INSTALL_BASE=$(INST_PREFIX)
+	perl Makefile.PL INSTALLDIRS=vendor
 
 test: build
 	$(MAKE) -f Makefile_Geo__GDAL test
